build.gradle

```diff
@@ -57,7 +57,7 @@ ext {
     jUnitVersion = '5.2.0'
     jacksonVersion = '2.9.6'
     jUnitPlatformVersion = '1.2.0'
-    ed25519Version = '1.3.3'
+    soraCryptoVersion = '8e4ace36a1'
```

Comment:
nice version of library) what will be next?)

Reply:
It is simply commit version. I can create a release and put tag here, actually.